packages/x-codemod/src/v6.0.0/text-props-to-localeText/expected-expression-values.spec.js

```diff
@@ -0,0 +1,271 @@
+      openNextView: custom_rightArrowButtonText
+    }} />
+
+  {/* DateTime */}
+  <DateTimePicker
+    localeText={{
+      cancelButtonLabel: custom_cancelText,
+      okButtonLabel: custom_okText,
+      end: custom_endText,
+      start: custom_startText,
+      clockLabelText: custom_getClockLabelText,
+      hoursClockNumberText: custom_getHoursClockNumberText,
+      minutesClockNumberText: custom_getMinutesClockNumberText,
+      secondsClockNumberText: custom_getSecondsClockNumberText,
+      calendarViewSwitchingButtonAriaLabel: custom_getViewSwitchingButtonText,
+      openPreviousView: custom_leftArrowButtonText,
+      previousMonth: custom_leftArrowButtonText,
+      openNextView: custom_rightArrowButtonText,
+      nextMonth: custom_rightArrowButtonText
+    }} />
+  <StaticDateTimePicker
+    localeText={{
+      cancelButtonLabel: custom_cancelText,
+      okButtonLabel: custom_okText,
+      end: custom_endText,
+      start: custom_startText,
+      clockLabelText: custom_getClockLabelText,
+      hoursClockNumberText: custom_getHoursClockNumberText,
+      minutesClockNumberText: custom_getMinutesClockNumberText,
+      secondsClockNumberText: custom_getSecondsClockNumberText,
+      calendarViewSwitchingButtonAriaLabel: custom_getViewSwitchingButtonText,
+      openPreviousView: custom_leftArrowButtonText,
+      previousMonth: custom_leftArrowButtonText,
+      openNextView: custom_rightArrowButtonText,
+      nextMonth: custom_rightArrowButtonText
+    }} />
+  <DesktopDateTimePicker
+    localeText={{
+      cancelButtonLabel: custom_cancelText,
+      okButtonLabel: custom_okText,
+      end: custom_endText,
+      start: custom_startText,
+      clockLabelText: custom_getClockLabelText,
+      hoursClockNumberText: custom_getHoursClockNumberText,
+      minutesClockNumberText: custom_getMinutesClockNumberText,
+      secondsClockNumberText: custom_getSecondsClockNumberText,
+      calendarViewSwitchingButtonAriaLabel: custom_getViewSwitchingButtonText,
+      openPreviousView: custom_leftArrowButtonText,
+      previousMonth: custom_leftArrowButtonText,
+      openNextView: custom_rightArrowButtonText,
+      nextMonth: custom_rightArrowButtonText
+    }} />
+  <MobileDateTimePicker
+    localeText={{
+      cancelButtonLabel: custom_cancelText,
+      okButtonLabel: custom_okText,
+      end: custom_endText,
+      start: custom_startText,
+      clockLabelText: custom_getClockLabelText,
+      hoursClockNumberText: custom_getHoursClockNumberText,
+      minutesClockNumberText: custom_getMinutesClockNumberText,
+      secondsClockNumberText: custom_getSecondsClockNumberText,
+      calendarViewSwitchingButtonAriaLabel: custom_getViewSwitchingButtonText,
+      openPreviousView: custom_leftArrowButtonText,
+      previousMonth: custom_leftArrowButtonText,
+      openNextView: custom_rightArrowButtonText,
+      nextMonth: custom_rightArrowButtonText
+    }} />
+
+  {/* Stand alone */}
+  <CalendarPicker
```

Comment:
I went to fast...
Those two components don't have a `localeText` prop.
We would need to add a `LocalizationProvider` around them.
Or to add support for `localeText`